authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,11 @@
-from django.conf.urls import url
+from django.urls import re_path
 
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
-    url(r'^users/?$', RegistrationAPIView.as_view()),
-    url(r'^users/login/?$', LoginAPIView.as_view()),
+    re_path('user/?$', UserRetrieveUpdateAPIView.as_view()),
+    re_path('users/?$', RegistrationAPIView.as_view()),
+    re_path('users/login/?$', LoginAPIView.as_view()),
```

Comment:
Why are we using `re_path` instead of `path`?

Reply:
**re_path()** allows me to write regex expressions in the URL. while **path()** does not allow that. and the URLs in authentication are regex

Reply:
Uniformity is key @reiosantos , please refactor the urls so that we use path throughout the application.